Etc/health_check.py

```
@@ -0,0 +1,71 @@
+CONSECUTIVE_FAILURES = 0
+ALERT_THRESHOLD = 5
+SLEEP_TIME = 60
+
+DATETIME_FORMAT = '%Y-%m-%d %H:%M:%S'
+
+def sendDiscordNotification(message):
+    payload = {
+        "content": message
+    }
+    try:
+        response = requests.post(WEBHOOK_URL, json=payload)
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)